server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java

```diff
@@ -36,18 +36,6 @@
       .noDefaultValue()
       .withDescription("Max memory of buffer manager for shuffle server");
 
-  public static final ConfigOption<Long> SERVER_BUFFER_SPILL_THRESHOLD = ConfigOptions
-      .key("rss.server.buffer.spill.threshold")
-      .longType()
```

Comment:
nit: checkValue. We have positiveLongValidator

Reply:
> nit: checkValue. We have positiveLongValidator
the config is removed